apps/cookbook/src/app/examples/styling-list-example/styling-list-example.component.html

```diff
@@ -0,0 +1,150 @@
+<h2>Unordered List Example</h2>
```

Comment:
Consider wrapping each of these examples in a card like we do for https://cookbook.kirby.design/#/home/showcase/badge, to separate them a little more

Reply:
We could also consider to make a 2 column layout for these examples, in addition to the above stuff.